bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -284,6 +284,9 @@ spec:
             - name: empty-dir
               mountPath: /opt/bitnami/keycloak/lib/quarkus
               subPath: app-quarkus-dir
+            - name: empty-dir
```

Comment:
We have a use case where we want to pre-populate our keycloak dockerfile image with existing data.
This particular mount over an existing data folder, basically gives us an empty folder, without our preload data once this is mounted.

can we have a values override added .e.g.?

            {{- if .Values.enableDataVolumeMount }}

we coded an example and were using it but would like this in future bitnami versions
![image](https://github.com/user-attachments/assets/fb24acd7-0eb6-4dcc-8474-5640a947fd31)
![image](https://github.com/user-attachments/assets/56efb6dc-eaa0-4577-806a-9950f892ca04)

Reply:
@JBKoerber could you please try to install the chart using the `values.yaml` below and let us know if that works for you? That should avoid the emptydir to ovewrite the data you pre-populated in the image.

```yaml
initContainers:
  - name: prepare-write-dirs
    image: "{{ template \"keycloak.image\" . }}"
    imagePullPolicy: "{{ .Values.image.pullPolicy }}"
    command:
      - /bin/bash
    args:
      - -ec
      - |
        . /opt/bitnami/scripts/liblog.sh

        info "Copying writable dirs to empty dir"
        # In order to not break the application functionality we need to make some
        # directories writable, so we need to copy it to an empty dir volume
        cp -r --preserve=mode /opt/bitnami/keycloak/data /emptydir/app-data-dir
        info "Copy operation completed"
    volumeMounts:
      - name: empty-dir
        mountPath: /emptydir
```

Reply:
@juan131 Thank you so much for the quick followup! It works, and  I was able to confirm using your recommended initContainers override code above.   It now has data/import files we were looking for using that approach.

Reply:
Thanks for confirming the workaround works @JBKoerber! I created [this PR](https://github.com/bitnami/charts/pull/28550) to adopt it by default so you don't need to use the custom values.